addon/templates/components/drag-sort-list.hbs

```diff
@@ -1,20 +1,43 @@
 {{#each items as |item index|}}
-  {{#drag-sort-item
-    additionalArgs                 = additionalArgs
-    item                           = item
-    index                          = index
-    items                          = items
-    group                          = group
-    handle                         = handle
-    class                          = childClass
-    tagName                        = childTagName
-    isHorizontal                   = isHorizontal
-    isRtl                          = isRtl
-    draggingEnabled                = draggingEnabled
-    dragEndAction                  = dragEndAction
-    determineForeignPositionAction = determineForeignPositionAction
-    sourceOnly                     = sourceOnly
-  }}
-    {{yield item index}}
-  {{/drag-sort-item}}
+  {{#if yieldDragSortItemComponent}}
+    {{yield (hash
+      data=item
+      index=index
+      Component=(component 'drag-sort-item'
```

Comment:
Capital 'C' to avoid naming collision?

Reply:
It’s what we had before. Capital letter because it’s a component.